### ISSUE TITLE
Slack

### ISSUE THREAD

**Issue by jonatanblue** [pact-foundation/roadmap#26]
Hey team!

The [community Slack](https://slack.pact.io) linked to from [the docs](https://docs.pact.io/help/how_to_ask_for_help) only allows user registrations for emails under the domains dius.com.au, smartbear.com and ae.com:
<img width="463" alt="image" src="https://github.com/pact-foundation/devrel/assets/2016004/aba7c676-cdae-4807-a6d2-04170bd14436">

Is this a mistake or have you restricted it for a reason? Is there another way to connect with the community?

**Comment by YOU54F** [comment]
Hey @jonatanblue 

that is really bizarre! I wouldn't have expected that restriction, and I'm not even sure what `ae.com`  is (American Eagle Apparel 😕 )

**Comment by YOU54F** [comment]
So this is bizarre!

This is how its setup

![image](https://github.com/pact-foundation/devrel/assets/19932401/41d58e2a-c390-4e35-a29e-398840ee7606)

I can see the restriction on emails

If I relax the restriction

<img width="923" alt="Screenshot 2023-09-15 at 13 32 41" src="https://github.com/pact-foundation/devrel/assets/19932401/5c8fc422-a0d3-4ad0-acb0-08fbb86ea0dd">

<img width="831" alt="Screenshot 2023-09-15 at 13 33 40" src="https://github.com/pact-foundation/devrel/assets/19932401/ccd818f6-5daf-49f8-9c44-eadb2b5ac8e9">

**Comment by YOU54F** [comment]
Dear Slack team - Please stop messing about with stuff that wasn't broken.

<img width="1804" alt="Screenshot 2023-09-15 at 13 35 39" src="https://github.com/pact-foundation/devrel/assets/19932401/d3426a0c-c495-43e5-81fc-aed56501f815">

**Comment by mefellows** [comment]
Looks like they've changed the way it works.

https://join.slack.com/t/pact-foundation/shared_invite/zt-238fesjgm-EYRYWpmv1zYZd58ByIGLmw

Does this work? Unfortunately it expires in 30 days. We might need to look
at the slackin invite again or something similar.

**Comment by YOU54F** [comment]
oh nice that one is working for me (in incognito mode)

<img width="862" alt="Screenshot 2023-09-15 at 13 39 10" src="https://github.com/pact-foundation/devrel/assets/19932401/5d44f89b-98e4-4ef7-ad6c-009a93fbd107">

<img width="905" alt="Screenshot 2023-09-15 at 13 39 19" src="https://github.com/pact-foundation/devrel/assets/19932401/d82c1c62-0eae-4b5e-bc6c-4477c7829d62">


some links from Slack

https://slack.com/intl/en-gb/help/articles/115004856503-Manage-how-people-join-your-workspace

https://slack.com/intl/en-gb/help/articles/201330256-Invite-new-members-to-your-workspace#request-an-invitation

**Comment by jonatanblue** [comment]
> Looks like they've changed the way it works.
> https://join.slack.com/t/pact-foundation/shared_invite/zt-238fesjgm-EYRYWpmv1zYZd58ByIGLmw

Yes, that one worked! Thanks :)

**Comment by YOU54F** [comment]
perpetual invite link

https://join.slack.com/t/pact-foundation/shared_invite/zt-23bi0zdz7-2sIA1omqUhz69MuD5yeZjw

have asked to get the redirect updated for slack.pact.io

Thanks for raising my friend, and helping test it out

**Comment by YOU54F** [comment]
Reopening until we have a confirmed permanent solution in place, and can ensure all the relevant places are updated.

There may be further restrictions on the quantity of users each link can support despite its validity period

**Comment by jonatanblue** [comment]
Good call. The [Slack docs](https://slack.com/intl/en-gb/help/articles/201330256-Invite-new-members-to-your-workspace#share-an-invitation-link) say this:

> Each invitation link can be used by up to 400 people.

So maybe the link above will only be valid for another 390 something users.

**Comment by mefellows** [comment]
Unfortunately the link has expired again. We're going to need to find a longer term solution for this.

**Comment by YOU54F** [comment]
so my last invite notification in slack stopped on the 16th, would be nice to sort a perma solution. thanks for updating Matt

**Comment by YOU54F** [comment]
had a report that the invite link is goosed again and restricted to specific email domains.

asked the gent to raise issue here so we can check its the same as listed and we can update the invite link again

**Comment by mefellows** [comment]
Thanks Yousaf - I've updated the link again.
